dashboard/app/controllers/api_controller.rb

```diff
@@ -200,7 +200,7 @@ def user_progress_for_stage
       end
     end
 
-    if level.finishable?
```

Comment:
My thinking had been that the case where we didn't have a `script_level` was one that we didn't really expect to happen outside of exceptional circumstances, and thus we were better off ignoring. Thinking about this some more, I think you might be right in that (a) there's no harm in logging and (b) if you go directly to a /level link and pass the level (for example when investigating an activity monitor failure) you likely want the slog even though you won't have a script_level. I'll update.